src/controllers.lisp

```diff
@@ -47,6 +49,10 @@
       (propagate-to-entity *sketch* x y #'on-right-click)
       (call-next-method))))
 
+(defmethod on-hover :around ((*sketch* sketch) x y)
+  (with-sketch (*sketch*)
```

Comment:
*draw-mode* nil missing - also, shouldn't we propagate-to-entity like everywhere else?